docs/sources/alerting/fundamentals/alert-rules/_index.md

```diff
@@ -16,27 +20,112 @@ weight: 100
 
 # Alert rules
 
-An alert rule is a set of evaluation criteria for when an alert rule should fire. An alert rule consists of one or more queries and expressions, a condition, and the duration over which the condition needs to be met to start firing.
+An alert rule is a set of evaluation criteria for when an alert rule should fire. An alert rule consists of one or more [queries and expressions, a condition][queries-and-conditions], and the duration over which the condition needs to be met to start firing.
```

Comment:
I would avoid in-line links here .. but if you think useful, feel free to leave!